ext/gd/gd.stub.php

```diff
@@ -52,13 +52,13 @@ function imagegrabscreen(): GdImage|false {}
 
 #endif
 
-function imagerotate(GdImage $im, float $angle, int $bgdcolor, int $ignoretransparent = 0): GdImage|false {}
+function imagerotate(GdImage $image, float $angle, int $bgdcolor, int $ignoretransparent = 0): GdImage|false {}
```

Comment:
Maybe just `$bgcolor` instead of `$bgdcolor`?

Reply:
Hrm, it looks like I missed to commit part of my changes :/ This was supposed to be `$background_color`/`$foreground_color`. Though if those are too long, I'm also fine with `$bg_color` and `$fg_color`.

Reply:
I'm fine with `$background_color`/`$foreground_color`.